src/OctoshiftCLI.Tests/bbs2gh/Handlers/GenerateScriptCommandHandlerTests.cs

```diff
@@ -344,7 +345,7 @@ public async Task Generated_Script_Contains_Exec_Function_Block()
     }
 
     [Fact]
-    public async Task One_Repo_With_Aws_Bucket_Name()
+    public async Task One_Repo_With_Aws_Bucket_Name_And_Region()
```

Comment:
Should we have a test which covers what happens if you don't submit a region? (I think we said that the region would be required for `generate-script`, but not `migrate-repo`.)

Reply:
In `generate-script` so far we have done very minimal validations the actual validations are going to happen in `migrate-repo` so there is no need to repeat them in `generate-script` too unless it's crucial to `generate-script` itself.

Reply:
Oh I see, it makes perfect sense to make region required if session token is passed! I will make that change in `migrate-repo` and add the validation. I still don't think we need to add extra validations in `generate-script` though.

Reply:
Addressed in 98de164b93e6c5ad0a9f24c52673cd80b4f9a7fc